setup.cfg

```diff
@@ -28,32 +20,74 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
-description = Example HEP files for testing and demonstrating
-license = BSD 3-clause
-long_description = file: README.md
-long_description_content_type = text/markdown
-url = https://github.com/scikit-hep/scikit-hep-testdata
+    Programming Language :: Python :: 3.9
+    Typing :: Typed
 keywords =
-  ROOT
-  HEP
-  testing
+    ROOT
+    HEP
+    testing
 
 [options]
+packages = find:
 install_requires =
     pyyaml
-    importlib_resources>=1.1
-    contextlib2; python_version<"3.3"
+    contextlib2;python_version<"3.3"
+    importlib_resources>=1.3;python_version<"3.9"
+    typing;python_version<"3.5"
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
 include_package_data = True
-zip_safe = True
-package_dir=
+package_dir =
     =src
-packages = find:
+zip_safe = True
 
 [options.packages.find]
 where = src
 
 [options.extras_require]
 test =
-    pytest >= 4.0.0, < 5.4.0
+    pytest>=4.0.0,<5.4.0
```

Comment:
Not sure why this constraint but not an issue for now, I guess.

Reply:
I have no idea why it's there. Python 3.10 will require a new (as yet unreleased) PyTest, so upper limits will be a problem soon.